lib/taskana-core/src/test/java/pro/taskana/ArchitectureTest.java

```diff
@@ -47,7 +47,7 @@ void apiClassesShouldNotDependOnInternalClasses() {
             .and()
             .haveSimpleNameNotEndingWith("BulkOperationResults")
```

Comment:
we could refactor BulkOperationResults into Interface + implementation in order to comply with this test.
Probably it would also be possible to restructure TaskanaHistoryEvent.
In addition, the test is disabled because there are violations in the report.api package. May be, we could refactor these classes to remove the violations